### DISCUSSION TITLE
Product Customization

### DISCUSSION THREAD

**Discussion by hparpinel** [eclipse-cdt/cdt#404] [Q&A]
Hi, I am new to Eclipse and I am a little bit lost on how to get started. I would like to customise my Eclipse CDT installation with my own logo and some specific environment features. I am not sure if I download the regular package or if there is an specific package to which I will need to add my features. 
Is there an specific forum where I can ask this questions or a detailed guide? 
Thank you in advance for all the help.

~H

**Comment by ruspl-afed** [comment]
I would recommend to start from [this](https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Fguide%2Fproduct_def.htm)
